snowpack/src/scan-imports.ts

```diff
@@ -265,14 +265,33 @@ export function scanDepList(depList: string[], cwd: string): InstallTarget[] {
     .reduce((flat, item) => flat.concat(item), []);
 }
 
+function filterObject(obj, predicate) {
+  let result = {};
+  for (const [key, value] of Object.entries(obj)) {
+    if (predicate(obj[key])) result[key] = value;
+  }
+  return result;
+}
+
+function findExtension(filePath, extensions) {
+  const baseExt = getExtension(filePath);
+  for (const extension of extensions) {
+    if (extension.toLowerCase() == baseExt) return true;
+  }
+  return false;
+}
+
 export async function scanImports(
   includeTests: boolean,
   config: SnowpackConfig,
 ): Promise<InstallTarget[]> {
   await initESModuleLexer;
+  const mountWithoutStatic = filterObject(config.mount, item => item.resolve);
+  const assetsExtensions = [".jpg", ".jpeg", ".png", ".gif"];
+  const filterAssets = (path, isDirectory) => !isDirectory && !findExtension(path, assetsExtensions);
```

Comment:
This currently filters out directories, is that the intention? Should it be instead:
```suggestion
  const filterAssets = (path, isDirectory) => isDirectory || !findExtension(path, assetsExtensions);
```